tests/encodeInputCapi.h

```diff
@@ -28,7 +28,7 @@ typedef void* EncodeOutputHandler;
 
 EncodeInputHandler createEncodeInput(const char * inputFileName, uint32_t fourcc, int width, int height);
 
-EncodeOutputHandler createEncodeOutput(const char * outputFileName, int width, int height);
+EncodeOutputHandler createEncodeOutput(const char* outputFileName, int width, int height, int fps = 0);
```

Comment:
I agree with you to modify my code to set a valid fps.
But I think I should replace
setUint32(header + 16, fps); // rate
with
setUint32(header + 16, fps <= 0 ? 30 : fps); // rate
That will just modify on one place.
But if I modify the format parameter fps = 30, there will be many places need to change.